### ISSUE TITLE
Status of this project?

### ISSUE THREAD

**Issue by philwinder** [kahing/goofys#684]
Hi @kahing and @dotslash,
I'm currently doing some work for a client of mine and some of it is historically using goofys. Whilst doing some new work I got stuck implementing a new provider, independent of goofys, because it is in the internal package.

I see you've been tinkering with the repo related to work with your employer in the past. Do you have any plans to continue development? Do you think you could open up the interfaces a bit more to make them useful outside of the core repo? Are you open to PRs? Can you open up control?

Basically, are there options other than forking?

Thanks, Phil

**Comment by dotslash** [comment]
> Whilst doing some new work I got stuck implementing a new provider, independent of goofys, because it is in the internal package.

what do you mean by a new provider? Can you send a PR that moves the functionality outside internal?
